_includes/tutorials/transforming/ksql/code/tutorial-steps/dev/outputs/transient-query/output-0.log

```diff
@@ -1,9 +1,9 @@
-+--------------------------------------------+--------------------------------------------+--------------------------------------------+--------------------------------------------+
-|ID                                          |TITLE                                       |YEAR                                        |GENRE                                       |
-+--------------------------------------------+--------------------------------------------+--------------------------------------------+--------------------------------------------+
-|294                                         |Die Hard                                    |1988                                        |action                                      |
-|354                                         |Tree of Life                                |2011                                        |drama                                       |
-|782                                         |A Walk in the Clouds                        |1995                                        |romance                                     |
-|128                                         |The Big Lebowski                            |1998                                        |comedy                                      |
```

Comment:
We just need to change the expected output in `tutorial-steps/dev/expected-transient.log` file to match this. This `output-0.log` is spurious